uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/htmleditor/HtmlEditor.java

```diff
@@ -99,35 +91,32 @@ public void onStartup(final ObservablePath path,
     @Override
     protected void loadContent() {
         baseView.hideBusyIndicator();
-        vfsServices.call(new RemoteCallback<String>() {
-            @Override
-            public void callback(final String htmlContent) {
-                editor.setContent(htmlContent);
-            }
+        htmlEditorService.call((String htmlContent) -> {
+            editor.setContent(htmlContent);
         }).readAllString(versionRecordManager.getCurrentPath());
     }
 
     @Override
     protected void save() {
         final String htmlContent = editor.getContent();
-        vfsServices.call(getSaveSuccessCallback(htmlContent.hashCode())).write(versionRecordManager.getCurrentPath(),
-                                                                               htmlContent);
+        htmlEditorService.call(getSaveSuccessCallback(htmlContent.hashCode())).write(versionRecordManager.getCurrentPath(),
```

Comment:
Could we please add simple unit test into `HtmlEditorTest`?

Reply:
Done.